languagetool-core/src/main/resources/org/languagetool/resource/spelling_global.txt

```diff
@@ -10061,4 +10061,1404 @@ Ben Ali
+Édouard Herriot
+Élisabeth Vigée Le Brun
+Élisée Reclus
+Émile Bernard
+Émile Littré
+Émilie du Châtelet
+Étienne Bonnot de Condillac
+Étienne de La Boétie
+Ismail Kadare
+Hélène Cixous
+Julio Cortázar
+Leonor Fini
+Julia Kristeva
+Tzvetan Todorov
+Alejo Carpentier
+Jiří Kolář
+Baron d'Holbach
+Marie d'Agoult
+Alfred Döblin
+Jean Arp
+Anton Reicha
+Isaac Casaubon
+Jean le Rond d'Alembert
+Peter Abelard
+Gaston Maspero
+Joseph-Louis Lagrange
+Jean-Marie Gustave Le Clézio
+Tahar Ben Jelloun
+André Grétry
+Theodore Beza
+Tristan Tzara
+Emil Cioran
+Léopold Sédar Senghor
+Peter Brook
+Austen Henry Layard
+George du Maurier
+Hilaire Belloc
+Bettina von Arnim
+Antoine Lavoisier
+Jacques Necker
+Giuseppe Garibaldi
+Eduard Mörike
+Gerhart Hauptmann
+Walter Benjamin
+Ernst Barlach
+Ernst Toller
+Lou Andreas-Salomé
+Adolf von Harnack
+Alexander Kluge
+Andreas Gryphius
+Artur Yusupov
+Carl Friedrich von Weizsäcker
+Edith Stein
+Ernst Jünger
+Friedrich Gottlieb Klopstock
+Friedrich Löchner
+Gottfried August Bürger
+Gustav Landauer
+Hans Magnus Enzensberger
+Heiner Müller
+Heinrich Cornelius Agrippa
+Heinrich Mann
+Helmut Krausser
+Hermann Sudermann
+Hrotsvitha
+Hugo Ball
+Jakob Böhme
+Jakob Wassermann
+Johann Gottfried Herder
+Johann Gottlieb Fichte
+Karl Rahner
+Kurt Schwitters
+Ludwig Meidner
+Manfred Eigen
+Matthias Deutschmann
+Max Klinger
+Moses Mendelssohn
+Nicholas of Cusa
+Ossi Weiner
+Oswald Spengler
+Otto of Freising
+Philipp Melanchthon
+Philipp Otto Runge
+Ricarda Huch
+Robert Hübner
+Rudolf Erich Raspe
+Siegfried Lenz
+Silvo Lahtela
+Stefan George
+Vladimir Budde
+Walter Jens
+Wilhelm Wundt
+Luděk Pachman
+Christa Wolf
+Victor Klemperer
+Wolf Biermann
+Walther Rathenau
+Erich Kästner
+Martin Walser
+Martin Opitz
+Herta Müller
+Wilhelm Ostwald
+Kurt Tucholsky
+Else Lasker-Schüler
+Kwame Nkrumah
+Johan Ludvig Runeberg
+Zachris Topelius
+Georg Büchner
+Alessandro Allori
+Carlo Collodi
+Edward Gibbon
+Edward Young
+Henry Fielding
+Horace Walpole
+James Macpherson
+John Gay
+John Wesley
+Joshua Reynolds
+Laurence Sterne
+Mary Wollstonecraft
+Matthew Gregory Lewis
+Philip Stanhope
+Samuel Johnson
+Thomas Gray
+Tobias Smollett
+William Congreve
+William Cowper
+Daniel Defoe
+Henry St. John
+Mary Andrea Garcia
+Samuel Richardson
+Ann Radcliffe
+Charles Lock Eastlake
+David Ricardo
+Frederick Marryat
+George Canning
+Henry Brougham
+Jane Austen
+John Bowring
+Leigh Hunt
+Percy Bysshe Shelley
+Robert Southey
+Samuel Taylor Coleridge
+Thomas Babington Macaulay
+Thomas Carlyle
+Thomas Robert Malthus
+Thomas Malthus
+William Beckford
+William Godwin
+William Wordsworth
+Giorgos Seferis
+Nikos Kazantzakis
+Odysseas Elytis
+Vasilios Kotronias
+Yannis Ritsos
+Eleftherios Venizelos
+Yelena Dembo
+Miguel Ángel Asturias Rosales
+Andreas Vesalius
+Gerardus Mercator
+John Amos Comenius
+Felix Dahn
+Athanasius Kircher
+Hans Jakob Christoffel von Grimmelshausen
+Johann Reuchlin
+Ulrich von Hutten
+Willibald Pirckheimer
+Klemens von Metternich
+Jan Hus
+Béla Balázs
```

Comment:
Hungarian names might benefit from being swapped around, since that's how Hungarians themselves do it and it's considered correct to also have `Balázs Béla`.